### DISCUSSION TITLE
Start pylance not installed messege

### DISCUSSION THREAD

**Discussion by Jakeo-93** [VSCodium/vscodium#1831] [General]
The Pylance extension is not installed but the python.languageServer value is set to "Pylance". Would you like to install the Pylance extension to use Pylance, or revert back to Jedi? 

getting the above warning messge even after pylance is installed. Any ideas on how to stop it. ? pretty sure pylance is working.

**Comment by daiyam** [comment]
Have you tried https://open-vsx.org/extension/detachhead/basedpyright? Thx
